README.md

```diff
@@ -35,9 +35,15 @@ lang := enry.GetLanguage("foo.cpp", "<cpp-code>")
 
 Developmemt
 -----------
-*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In oreder to update to latest upstream run
+*enry* re-uses parts of original [linguist](https://github.com/github/linguist)  to generate internal data structures. In order to update to latest upstream and generate the necessary code you must run:
 
-    make clean code-generate
+    go generate
+
+We update enry due to changes  in linguist's master branch related to the following files:
```

Comment:
We don't have a procedure to detect automatically changes in linguist and sync in fact, it happens when during coding some new functionality of fixing something we run go generate.